CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - _...Add new stuff here..._
 
 ### 🐞 Bug fixes
+- Remove extra "the"s in the documentation [#922](https://github.com/maplibre/maplibre-style-spec/pull/922)
```

Comment:
Not sure it is interesting enough for a changelog entry, I would consider removing...